beacon-chain/powchain/service.go

```diff
@@ -284,6 +288,32 @@ func (w *Web3Service) handleDelayTicker() {
 		log.Error(err)
 	}
 }
+func (w *Web3Service) rerunRPCClientAndSubscribe(headSub ethereum.Subscription, header *gethTypes.Header) {
+	rpcClient, err := gethRPC.Dial(w.endpoint)
+	if err != nil {
+		log.Errorf("Access to PoW chain is required for validator. Unable to connect to Geth node: %v", err)
```

Comment:
Shouldn't you set the `w.runError` and return?

Reply:
yes